api/integrations/lead_tracking/hubspot/lead_tracker.py

```diff
@@ -119,7 +119,7 @@ def update_company_active_subscription(
     def _get_or_create_company_by_domain(self, domain: str) -> dict:
         company = self.client.get_company_by_domain(domain)
         if not company:
-            company = self.client.create_company(name=domain)
+            company = self.client.create_company(name=domain, domain=domain)
```

Comment:
Since the domain param is passed twice (which looks weird) this could benefit from a short comment I think.

Reply:
Done ✅